test/units/galaxy/test_collection.py

```diff
@@ -40,9 +40,9 @@ def reset_cli_args():
 
 
 @pytest.fixture()
-def collection_input(tmp_path_factory):
+def collection_input(tmp_path_factory, path_sufix='test-ÅÑŚÌβŁÈ Collections Input'):
```

Comment:
I think "suffix" is spelled with double "f", we should probably fix that.

Reply:
done